File: JavaNIDE/volume/app/src/main/java/sample/For4.java
package sample;
import java.util.*;
 class For4
 {
  static int i,j,n,fact=1;
  static float sum=0,p,z;
  public static void main(String[] arr)
  {
    Scanner m=new Scanner(System.in);
    System.out.println("enter no");
    n=m.nextInt();
   for(i=1;i<=n;i++)
   {
    System.out.print(i+"*"+i+"/"+i+"+"+(i+1)+"!");
    System.out.print(" + ");
    for(j=1;j<=i+1;j++)
    {
     fact=fact*j;
    }
    z=(float)(i*i);
    p=(float)(i+fact);
    sum=sum+z/p;
    fact=1;
   }
   System.out.print("\nsum is"+sum);
  }
 
  
}

File: CodingC/for4.c
#include<stdio.h>

int main()
{
 float i,j,m=0,fact=1,x=0;
for(i=1;i<=3;i++)
{
printf("%.0f!/",i);
 for(j=1;j<=i;j++)
{
  fact=fact*j;
printf("(%.0f)^3",j);
if(j<i)
{
printf("+");
}
m=m+j*j*j;
}
printf("  +  ");
x=x+fact/m;
fact=1;
m=0;
}
printf("sum of the series is %.2f",x);
}
